hbase-asyncfs/src/main/java/org/apache/hadoop/hbase/io/asyncfs/FanOutOneBlockAsyncDFSOutputHelper.java

```diff
@@ -383,7 +400,7 @@ private static List<Future<Channel>> connectToDataNodes(Configuration conf, DFSC
     BlockConstructionStage stage, DataChecksum summer, EventLoopGroup eventLoopGroup,
     Class<? extends Channel> channelClass) {
     StorageType[] storageTypes = locatedBlock.getStorageTypes();
-    DatanodeInfo[] datanodeInfos = locatedBlock.getLocations();
+    DatanodeInfo[] datanodeInfos = getLocatedBlockLocations(locatedBlock);
```

Comment:
There's another place to be fixed. :)
https://github.com/apache/hbase/blob/84ccae368f6ed94a751d576c59fc695c683f1c8b/hbase-asyncfs/src/main/java/org/apache/hadoop/hbase/io/asyncfs/FanOutOneBlockAsyncDFSOutputHelper.java#L498

Reply:
Done.